### ISSUE TITLE
command line options should override configuration file

### ISSUE THREAD

**Issue by faxm0dem** [collectd/collectd#366]
Example:

```
# /usr/sbin/collectd -C /etc/collectd.conf -P /tmp/collectd.pid
[...]
fopen (/var/run/collectd.pid): Permission denied
```

**Comment by rubenk** [comment]
Absolutely.

**Comment by tokkee** [comment]
This was originally fixed in 09fb81e130ad70bf0547cec3d3c349ce8cb092cf.

I have an imho nicer solution in #1750
